src/assets/css/themes/darkly-compact.scss

```diff
@@ -56,4 +57,6 @@ hr.my-3 {
   }
 }
 
-@import "../../../../node_modules/bootstrap/scss/bootstrap";
+.btn-outline-secondary {
+  color: $gray-500;
+}
```

Comment:
If there's a way to do this by setting an scss variable, let me know. That is preferable to needing to put this selector in every darkly scss file.

Reply:
I think there may be; one sec.

Reply:
This is, I think, a consequence of the way we're doing these theme colors, the concern I brought up [here](https://github.com/LemmyNet/lemmy-ui/pull/1682#issuecomment-1612271503).

In litely, `secondary` is green -- the "primary" accent color. In darkly, `secondary` is a a very dark gray. So using the same class -- `btn-outline-secondary` -- results in unpredictably different outcomes. We're using `secondary` to mean different things in different themes.

I think maybe the best solution for now (other than revisiting the way we use `secondary`) would be to do this:

`_variables.darkly.scss`:
```scss
$secondary: $gray-600;
```

It might also be worth considering adding the `text-dark` class to the *non-active* radio toggles so that the text has a higher contrast than the borders. This would make the litely ones have dark gray text instead of green text, however -- another way the inconsistent theme color naming would nail us.

Reply:
I'm also just noticing that having `$input-border-color: $body-bg` in darkly is kind of an odd choice:

<img width="624" alt="Screenshot 2023-07-04 at 12 58 40 PM" src="https://github.com/LemmyNet/lemmy-ui/assets/643417/ffa9b6c5-4a44-455f-8d9a-23e266c4bee1">